Examples/VaporExample/Resources/Views/index.leaf

```diff
@@ -14,6 +14,8 @@
       font-family: "Trebuchet MS", Helvetica, sans-serif;
     }
   </style>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <script src="../bson.bundle.js"></script>
```

Comment:
I assume there's nowhere this is hosted publicly like jquery is to avoid having to check it into the repo?